extensions/ql-vscode/src/common/commands.ts

```diff
@@ -1,4 +1,5 @@
 import { CommandManager } from "../packages/commands";
+import type { Uri } from "vscode";
```

Comment:
I'd have preferred to not import from `vscode` in the `src/common` directory, but importing a type should be fine. We could also create an interface that defines the necessary properties to avoid importing from `vscode` here, but we'd need to find out which methods on `Uri` in command arguments are being used. For this command, it's just `fsPath`, but for others there may be more.